r/tests/testthat/helper-data.R

```diff
@@ -68,9 +68,13 @@ make_big_string <- function() {
   rep(purrr::map_chr(2047:2050, ~paste(sample(letters, ., replace = TRUE), collapse = "")), 2^18)
 }
 
-make_string_of_size <- function(size = 1) {
+make_random_string_of_size <- function(size = 1) {
   purrr::map_chr(1000*size, ~paste(sample(letters, ., replace = TRUE), collapse = ""))
 }
 
+make_string_of_size <- function(size = 1) {
+  purrr::map_chr(1000*size, ~paste(rep(letters, length = .), collapse = ""))
```

Comment:
It's not immediately clear to me why this can't just be:
```r
paste(rep(letters, length = 1000*size), collapse = "")
```

Reply:
Oh, I see—because size can be a vector with length > 1

Reply:
I was lazily copying and pasting, but I've updated it